packages/xwayland.rb

```diff
@@ -3,95 +3,51 @@
-    url_patch1 = "https://chromium.googlesource.com/chromiumos/overlays/chromiumos-overlay/+/067ac4b5060c16e6687a97cbb4bcdbaf5a0b5639/x11-base/xwayland/files/0001-Eliminate-conflict-with-X11-Xlib.h-with-khronos-eglp.patch?format=TEXT"
-    uri_patch1 = URI.parse url_patch1
-    filename_patch1 = 'patch1_base64'
-    sha256sum_patch1 = '16f5d5d1986daf07b4e0f01eb0e68b53d8ef301d169f3cc9493158c8b4987f16'
-    puts "Downloading patch1".yellow
-    system('curl', '-s', '--insecure', '-L', '-#', url_patch1, '-o', filename_patch1)
-    abort 'Checksum mismatch. :/ Try again.'.lightred unless
-      Digest::SHA256.hexdigest( File.read( filename_patch1 ) ) == sha256sum_patch1
-    puts "patch1 archive downloaded".lightgreen
-    system 'base64 --decode patch1_base64 > patch1'
-    system 'patch -p 1 < patch1'
-    url_patch2 = "https://chromium.googlesource.com/chromiumos/overlays/chromiumos-overlay/+/067ac4b5060c16e6687a97cbb4bcdbaf5a0b5639/x11-base/xwayland/files/0001-xwayland-Fall-back-to-gbm_bo_create-if-no-modifiers-.patch?format=TEXT"
-    uri_patch2 = URI.parse url_patch2
-    filename_patch2 = 'patch2_base64'
-    sha256sum_patch2 = 'd44bfca0e26ca06892b37b12faca2a6756cdc2455aac36ac7607bfa188856b35'
-    puts "Downloading patch2".yellow
-    system('curl', '-s', '--insecure', '-L', '-#', url_patch2, '-o', filename_patch2)
-    abort 'Checksum mismatch. :/ Try again.'.lightred unless
-      Digest::SHA256.hexdigest( File.read( filename_patch2 ) ) == sha256sum_patch2
-    puts "patch2 archive downloaded".lightgreen
-    system 'base64 --decode patch2_base64 > patch2'
-    system 'patch -p 1 < patch2'
-    
-    url_patch3 = "https://chromium.googlesource.com/chromiumos/overlays/chromiumos-overlay/+/067ac4b5060c16e6687a97cbb4bcdbaf5a0b5639/x11-base/xwayland/files/0001-xwayland-sysmacros.patch?format=TEXT"
-    uri_patch3 = URI.parse url_patch3
-    filename_patch3 = 'patch3_base64'
-    sha256sum_patch3 = 'b66eb66cc8023ab2170db91e4d6038dca99b814addfcb0977d30479a2e514ab8'
-    puts "Downloading patch3".yellow
-    system('curl', '-s', '--insecure', '-L', '-#', url_patch3, '-o', filename_patch3)
-    abort 'Checksum mismatch. :/ Try again.'.lightred unless
-      Digest::SHA256.hexdigest( File.read( filename_patch3 ) ) == sha256sum_patch3
-    puts "patch3 archive downloaded".lightgreen
-    system 'base64 --decode patch3_base64 > patch3'
-    system 'patch -F 10 --force -p 1 < patch3'
-  end
+  depends_on 'pixman'
+  depends_on 'wayland'
+  depends_on 'xkbcomp'
+  depends_on 'xorg_lib'
 
   case ARCH
-    when 'armv7l', 'aarch64'
-      PEER_CMD_PREFIX='/lib/ld-linux-armhf.so.3'
-    when 'i686'
-      PEER_CMD_PREFIX='/lib/ld-linux-i686.so.2'
-    when 'x86_64'
-      PEER_CMD_PREFIX='/lib64/ld-linux-x86-64.so.2'
+  when 'armv7l', 'aarch64'
+    PEER_CMD_PREFIX = '/lib/ld-linux-armhf.so.3'.freeze
+  when 'i686'
+    PEER_CMD_PREFIX = '/lib/ld-linux-i686.so.2'.freeze
+  when 'x86_64'
+    PEER_CMD_PREFIX = '/lib64/ld-linux-x86-64.so.2'.freeze
```

Comment:
Not sure why we need to `.freeze`.  It looks like rubocop is a bit too authoritarian. :)